src/services/sources.ts

```diff
@@ -53,7 +52,7 @@ export const sourcesApi = createApi({
         method: 'DELETE',
       }),
     }),
-    updateSource: builder.mutation<void, SourceWithRepository>({
+    updateSource: builder.mutation<any, SourceWithRepository>({
```

Comment:
[It](https://docs.testkube.io/openapi/#tag/test-sources/operation/updateTestSource) returns an updated `SourceWithRepository`, no? There was `void` because we did not use this updated `SourceWithRepository` after the request.

Reply:
It doesn't - it returns partial of `spec` and some changes. Didn't want to try to type it, because I'm not sure if it's expected for that.

Reply:
OK, so we can't relay on docs all the time 👌